DQM/HcalCommon/interface/Constants.h

```diff
@@ -215,6 +215,42 @@ namespace hcaldqm
 		int const RAW_EMPTY = 16;
 		int const UTCA_DATAFLAVOR = 0x5;
 
+		double const adc2fC[256] = {
```

Comment:
dqm doesn't seem like the place we should be defining adc2fC. I guess this is already done elsewhere?